infrastructure/containerd/repo_test.go

```diff
@@ -59,11 +72,10 @@ func TestMicroVMRepo_Integration(t *testing.T) {
 	Expect(err).NotTo(HaveOccurred())
 	Expect(exists).To(BeFalse())
 
-	exists, err = repo.Exists(ctx, testOwnerName, testOwnerNamespace)
-	Expect(err).NotTo(HaveOccurred())
-	Expect(exists).To(BeFalse())
```

Comment:
Removed because it  was a duplication, the previous 3 lines are exactly the same.